adb/adb_protocol.py

```diff
@@ -61,7 +78,7 @@ def MakeWireIDs(ids):
       cmd_id: sum(c << (i * 8) for i, c in enumerate(bytearray(cmd_id)))
       for cmd_id in ids
   }
-  wire_to_id = {wire: cmd_id for cmd_id, wire in id_to_wire.items()}
+  wire_to_id = {wire: cmd_id for cmd_id, wire in iteritems(id_to_wire)}
```

Comment:
why go from `items` to `iteritems`? This isn't a big object so the overhead of the copy is small, and it's correct in py 3 anyway

Reply:
You're right, no real benefit to iterates here. Reverted to items()